src/utils/disable-download.ts

```diff
@@ -0,0 +1,11 @@
+import semver from 'semver';
+import { isJest } from './is-jest';
+
+export function disableDownload(version: string) {
```

Comment:
Let's add a JSDoc for this.

Reply:
Added JSDocs with a link to the apple silicon version 11.0.0 release blog.